osidb_bindings/templates/endpoint_module.py.jinja

```diff
@@ -111,4 +111,12 @@ def sync(
     return sync_detailed(
         {{ kwargs(endpoint) }}
     ).parsed
+
+QUERY_PARAMS = {
```

Comment:
can we add this to tutorial and changelog as well ?

Reply:
We technically can, but this is the part of the low level bindings, I would rather expose that in some nice way via the session maybe or some helper on the surface so the user does not need to go deep inside the bindings logic.

Reply:
fine with me ;)